.github/workflows/build-test-release.yml

```
@@ -198,7 +199,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        splunk-version: [8.1, 8.2]
+#        splunk-version: [8.1, 8.2]
+        splunk-version: [8.2]
```

probably we should test with 3 last splunk versions
```suggestion
        splunk-version: [8.1, 8.2, 9.0]
```